### PR TITLE
[hailctl] Initial hailctl commit

### PR DESCRIPTION
**Pull request by tpoterba**
Add `hailctl` as a top-level project. Mostly a copy of cloudtools,
with some additional plumbing on top.

### CODE REVIEW

**Comment by jigold** [comment]
Should we delete the cloudtools folder from hail? Although we should check there isn't anything else important in there first.

**Comment by tpoterba** [comment]
yes, that's going to be the next PR on this.

File: hailctl/python/hailctl/__main__.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        print_help()
+        sys.exit(0)
+    else:
+        main_module = sys.argv[1]
+        args = sys.argv[2:]
+        mod = jmp.get(main_module)
+        if not mod:
+            # no module by this name
+            print_help()
+            sys.exit(0)
```
Comment (jigold) [inline hailctl/python/hailctl/__main__.py]: Should the exit codes here be 0 or 1?

Comment (tpoterba) [reply hailctl/python/hailctl/__main__.py]: --help usually returns exit code 0, and these are just shortcuts for that

Comment (tpoterba) [reply hailctl/python/hailctl/__main__.py]: this was a good question though, I looked at some other tools to answer

**Pull request merged**